src/components/track-changes/TrackChangesPanel.tsx

```diff
@@ -65,7 +48,11 @@ export function TrackChangesPanel() {
         ids.push(child.id)
       })
     }
-    execCmd(trackCommands.setChangeStatuses(CHANGE_STATUS.accepted, ids))
+    editor.view &&
+      trackCommands.setChangeStatuses(CHANGE_STATUS.accepted, ids)(
```

Comment:
as mentioned previously execCmd was a nice abstraction, it can either leave in a hook of its own or be created in a QB source.

Reply:
updated to hook